build/syntax-highlight.ts

```diff
@@ -95,7 +92,7 @@ export function syntaxHighlight($, doc) {
     if (!match) {
       return;
     }
-    const name = ALIASES.get(match[1]) || match[1];
+    const name = match[1].replace("-nolint", "");
```

Comment:
`ALIASES` is still used for sh/shell though.

Reply:
Oops, didn't mean to remove aliases support, fixed!